lib/v2/screens/explore/interactor/viewmodels/events.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter/material.dart';
+import 'package:meta/meta.dart';
+
+/// --- EVENTS
+@immutable
+abstract class ExploreEvent extends Equatable {
+  @override
+  List<Object> get props => [];
+}
+
+class LoadExplore extends ExploreEvent {
+  final String userName;
+
+  @override
+  String toString() => 'LoadExplore';
```

Comment:
You could parse the userName value into the string, Example:

String toString() => 'LoadExplore: { userName: $userName }'

In order to watch the value in console debug by the BlocObserver 😃

Reply:
yes! Done

Reply:
In fact .. where is the constructor this param must be required (jaja I didnt notice the last time).

` const LoadExplore({@required this.userName});`

and Idk if the assert version is needed too.

` const LoadExplore({@required this.userName}):assert(userName!=null);`